packages/babel-parser/test/fixtures/typescript/type-alias/union-intersection/input.ts

```diff
@@ -0,0 +1,7 @@
+type J = number | string
```

Comment:
Can we just augment:

https://github.com/babel/babel/blob/main/packages/babel-parser/test/fixtures/typescript/types/union-intersection/input.ts

instead?